taskcluster/adhoc_taskgraph/worker_types.py

```diff
@@ -51,24 +51,28 @@ def build_scriptworker_signing_payload(config, task, task_def):
         "maxRunTime": worker["max-run-time"],
         "upstreamArtifacts": worker["upstream-artifacts"],
     }
-    if worker.get("mac-behavior"):
-        task_def["payload"]["behavior"] = worker["mac-behavior"]
-        if worker.get("product"):
-            task_def["payload"]["product"] = worker["product"]
-        if worker.get("entitlements-url"):
-            task_def["payload"]["entitlements-url"] = worker["entitlements-url"]
-        if worker.get("provisioning-profile-url"):
-            task_def["payload"]["provisioning-profile-url"] = worker["provisioning-profile-url"]
+
+    for key in (
+        "behavior",
+        "product",
+        "entitlements-url",
+        "provisioning-profile-url",
+        "loginitems-entitlements-url",
+        "nativemessaging-entitlements-url",
+        "provisioning-profile-url",
+    ):
+        if worker.get(key):
+            task_def["payload"][key] = worker[key]
```

Comment:
By keying on `mac-behavior`, we know that it's mac. I think as currently written we'll copy these things in even if it's a windows signing request?